include/FreeRTOS_IP.h

```diff
@@ -126,6 +126,15 @@
     #define ipICMP_NEIGHBOR_SOLICITATION_IPv6          ( ( uint8_t ) 135U )
     #define ipICMP_NEIGHBOR_ADVERTISEMENT_IPv6         ( ( uint8_t ) 136U )
 
+    #define ipIPv6_EXT_HEADER_HOP_BY_HOP               0U
+    #define ipIPv6_EXT_HEADER_DESTINATION_OPTIONS      60U
+    #define ipIPv6_EXT_HEADER_ROUTING_HEADER           43U
+    #define ipIPv6_EXT_HEADER_FRAGMENT_HEADER          44U
+    #define ipIPv6_EXT_HEADER_AUTHEN_HEADER            51U
+    #define ipIPv6_EXT_HEADER_SECURE_PAYLOAD           50U
+    /* Destination options may follow here in case there are no routing options. */
+    #define ipIPv6_EXT_HEADER_MOBILITY_HEADER          135U
+
```

Comment:
I did find a couple more options here - we might want to add them as well for completeness.
`Host Identity Protocol - 139`
`Shim6 Protocol - 140`.

What do you think @htibosch?

Reply:
I had also seen them but let's stick to RFC 8200 for the moment.